src/main/java/se/sundsvall/seabloader/service/InvoiceService.java

```diff
@@ -70,6 +71,10 @@ private void sendInvoiceToInvoiceCache(final List<InvoiceId> invoiceIds) {
 						invoiceCacheClient.sendInvoice(invoicePdfRequest);
 						invoiceEntity.setStatus(PROCESSED);
 						invoiceRepository.save(invoiceEntity);
+					} catch (ThrowableProblem tb) {
```

Comment:
What is the difference with the catch block below (except the log message)?

Reply:
No difference more than avoiding an extra log-message. I'll change it to only catch Exception.